std/haxe/macro/Compiler.hx

```diff
@@ -397,7 +397,7 @@ class Compiler {
 	}
 
 	/**
-		Register a custom medatada for documentation and completion purposes
```

Comment:
lol metatada...

Reply:
My hands keep making that one, have to type it 2 or 3 times to get it right x_x